### DISCUSSION TITLE
Favicon Not Rendering

### DISCUSSION THREAD

**Discussion by oleersoy** [community/community#21743] [Other Feature Feedback, Questions, & Ideas]
<p>I have a favicon in this repository that I saved with GIMP as a Microsoft Application Icon:</p>
<p><a href="https://github.com/fireflysemantics/fireflysemantics.com/tree/gh-pages" rel="nofollow noopener">https://github.com/fireflysemantics/fireflysemantics.com/tree/gh-pages</a></p>
<p>However when I load the page the icon is not rendering:</p>
<p><a href="https://fireflysemantics.com/" rel="nofollow noopener">https://fireflysemantics.com/</a></p>
<p>Any ideas?</p>
<p>TIA,</p>
<p>Ole</p>

**Comment by Gisgar3** [comment, marked as answer]
<p>I checked the favicon pushed into the repository and checked the  <strong>GitHub Page</strong> you shared and the favicon infact <em>is</em> loading correctly; completely visible within the browser tabs.</p>

**Comment by oleersoy** [comment]
<p>Wow - That’s weird - I’ve been checking it for the last couple of days with Cognito windows and nothing - Looks like you’re the magic touch - Thanks!</p>
